src/main/java/com/wuest/prefab/structures/render/StructureRenderHandler.java

```diff
@@ -195,7 +196,7 @@ private static void renderBlockModel(BlockRenderView world, BakedModel bakedMode
 
     private static void renderBlocks(VertexBuffer vertexBuffer, BlockPos pos, Vec3d cameraPos, MatrixStack matrixStack) {
         matrixStack.push();
-        matrixStack.translate(pos.getX()-cameraPos.getX(), pos.getY()-cameraPos.getY(), pos.getZ()-cameraPos.getZ());
+        matrixStack.translate(pos.getX()-cameraPos.getX(), pos.getY()-cameraPos.getY()+0.001, pos.getZ()-cameraPos.getZ());
```

Comment:
Needs extra spaces around the math operations in this method. They are difficult to notice otherwise.

If you are using IntelliJ and used the default keyboard layout you can press Alt + Ctrl + L to format the document.